docs/content/ingestion/data-formats.md

```diff
@@ -90,9 +99,43 @@ Since the CSV data cannot contain the column names (no header is allowed), these
     }
   }
 ```
+
+The `columns` field must match the columns of your input data in the same order. 
+
 Be sure to change the `delimiter` to the appropriate delimiter for your data. Like CSV, you must specify the columns and which subset of the columns you want indexed.
 
+### Regex
+```json
+  "parseSpec":{
```

Comment:
This should either let you provide `columns` or it should document what the columns are going to be called. I'm pretty sure this example as written could not work, because the columns will be called "column_1", "column_2" etc and not "page", "language", etc